onsp_co2/models/carbon_factor.py

```diff
@@ -25,6 +25,15 @@ class CarbonFactor(models.Model):
     child_ids = fields.One2many("carbon.factor", "parent_id")
     child_qty = fields.Integer(compute="_compute_child_qty")
     value_ids = fields.One2many("carbon.factor.value", "factor_id")
+    
+    chart_of_account_ids = fields.One2many("account.account", "carbon_in_factor_id")
+    chart_of_account_qty = fields.Integer(compute="_compute_chart_of_account_qty")
+    
+    product_ids = fields.One2many("product.template", "carbon_in_factor_id")
+    product_qty = fields.Integer(compute="_compute_product_qty")
+    
+    product_categ_ids = fields.One2many("product.category", "carbon_in_factor_id")
+    product_categ_qty = fields.Integer(compute="_compute_product_categ_qty")
```

Comment:
What about `out` factors?  You can't ignore them

Reply:
I totally forgot about them. In the new version I used read_group and also take `out` factor